api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java

```diff
@@ -274,7 +274,7 @@ public CohortDefinition getPatientsWhoInitiatedTarvAtThisFacilityDuringCurrentMo
     cd.addParameter(new Parameter("onOrAfter", "Start Date", Date.class));
     cd.addParameter(new Parameter("onOrBefore", "End Date", Date.class));
     cd.addParameter(new Parameter("location", "Location", Location.class));
-    cd.setB10Flag(new Boolean("false"));
+    cd.setB10Flag(new Boolean("true"));
```

Comment:
What is the effect of this change to other queries, say resumo mensal dataset? we are using the same, and it would be great if we test if we are returning the same values, especially based on what was run last